README.md

```diff
@@ -4,7 +4,7 @@ microsoft.github.io
 As Microsoft's open source presence and volume of released source code up on GitHub has
 increased, we've reimagined the portal and index for our repos.
 
-Today you can find, filter and review all sorts of open source up at https://opensource.microsoft.com.
+Today you can find, filter and review all sorts of open source up at https://opensource.microsoft.com / https://open.microsoft.com.
```

Comment:
the site you added is the community links which is not the same as opensource.microsoft.com. What you could do is add a link to the page wrapping https://opensource.microsoft.com so it is clickable